admin_guide/scheduler.adoc

```diff
@@ -668,3 +668,163 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master
 ----
 endif::[]
+
+[[controlling-pod-placement-projects]]
+=== Control Pod Placement to Projects
+
+The _Pod Node Selector_ admission controller allows you to force pods onto nodes associated with a specific project and prevent pods from being scheduled in those nodes.
+
+The *Pod Node Selector* admission controller determines where a pod can be placed using xref:../../architecture/core_concepts/pods_and_services.adoc#labels[labels on projects] and node selectors specified in pods. A new pod will be placed on a node associated with a project  only if the node selectors in the pod match the labels in the project.
+
+After the pod is created, the node selectors are merged into the pod so that the pod specification includes the labels originally included in the specification and any new labels from the node selectors. The example below illustrates the merging effect.
+
+The *Pod Node Selector* admission controller also allows you to create a list of labels that are permitted in a specific project. This list acts as a _whitelist_ that lets developers know what labels are acceptable to use in a project and gives administrators greater control over labeling in a cluster.
+
+To activate the *Pod Node Selector* admission controller: 
+
+. Configure the *Pod Node Selector* admission controller and whitelist, using one of the following methods: 
+
+** Add the following to the master configuration file (*_/etc/origin/master/master-config.yaml_*):
++
+----
+admissionConfig:
+  pluginConfig:
+    PodNodeSelector:
+      configuration:
+        podNodeSelectorPluginConfig: <1>
+          clusterDefaultNodeSelector: "k3=v3" <2>
+          ns1: region=west,env=test,infra=fedora,os=fedora <2>
+----
++
+<1> Adds the *Pod Node Selector* admission controller plug-in.
+<2> Creates default labels for all nodes.
+<3> Creates a whitelist of permitted labels in the specified project. Here, the project is `ns1` and the labels are the `key=value` pairs that follow.
+
+* Create a file containing the admission controller information:
```

Comment:
Should this be a double ** as well? Is this the alternate to the above? (that's talked about as "one of the following methods")